specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5818,7 +5819,8 @@
       "required": true,
       "type": "string",
       "description": "The resource scope.",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "method",
+      "x-ms-skip-url-encoding": true
```

Comment:
why skipping encoding?

Reply:
As mentioned in description, the url encoding will fail a cli playback test, cli-test-framework will unable to detect and replace the subscription: https://github.com/Azure/azure-rest-api-specs/issues/9270